package.json

```diff
@@ -313,5 +313,13 @@
       "prettier --write"
     ],
     "*.{md,json,yaml,yml}": "prettier --write"
-  }
+  },
+  "sideEffects": [
+    "*.css",
+    "*.scss",
+    "src/shortcuts/index.ts",
+    "src/shortcuts/default.ts",
```

Comment:
I added these paths so that you can use the shortcuts in the playground. 
When you start use this shortcuts in demo components you can delete this 2 lines